loader/windows/icd_windows_hkr.c

```diff
@@ -17,8 +17,9 @@
  */
 
 #include "icd.h"
-#include "icd_windows_hkr.h"
 #include <windows.h>
+#include "icd_windows_hkr.h"
```

Comment:
We fixed the include file dependencies in #79, correct?  If so, let's keep the #include for <windows.h> where it was.

Reply:
Fixed.